egs2/gtsinger/svs1/conf/tuning/train_xiaoice.yaml

```diff
@@ -0,0 +1,72 @@
+
+
+##########################################################
+#                  SVS MODEL SETTING                     #
+##########################################################
```

Comment:
Please reduce the configurations (only keep the one that you used for the model training)/